app/src/androidTest/java/ch/sdp/vibester/BuzzerScoreUpdaterTest.kt

```diff
@@ -24,29 +24,59 @@ public class BuzzerScoreUpdaterTest {
         val scoreArray = arrayOf(0, 0, 0, 0)
         val testUpdater = BuzzerScoreUpdater(idArray, scoreArray)
         for (id in idArray) {
-            testUpdater.updateScoresArray(id, 1)
+            testUpdater.updateScoresArray(id, true)
             assertTrue(testUpdater.getMap()[id] == 1)
         }
     }
 
     @Test
-    fun getWinnerIdTest() {
+    fun arrayUpdateWithWrongIdReturns() {
         val idArray = arrayListOf(R.id.buzzer_0, R.id.buzzer_1, R.id.buzzer_2, R.id.buzzer_3)
         val scoreArray = arrayOf(0, 0, 0, 0)
         val testUpdater = BuzzerScoreUpdater(idArray, scoreArray)
-        assertTrue(testUpdater.getWinnerId() == -1)
-        testUpdater.updateScoresArray(idArray[0], 1)
-        assertTrue(testUpdater.getWinnerId() == idArray[0])
+        testUpdater.updateScoresArray(-1, true)
+        for (id in idArray) {
+            assertTrue(testUpdater.getMap()[id] == 0)

```

Comment:
```suggestion
            assertEquals(0, testUpdater.getMap()[id])
```
It is maybe more precise to use assertEqual. 
Indeed if there is an error, it will be able to say: Value received, Value Expected.
Whereas in your case it will just say, received false whereas expecting true